definitions/divorce/json/CaseEvent.json

```diff
@@ -1138,7 +1138,7 @@
     "PostConditionState": "AosCompleted",
     "CallBackURLAboutToSubmitEvent": "${CCD_DEF_COS_URL}/aos-received",
     "RetriesTimeoutURLAboutToSubmitEvent": 120120,
-    "CallBackURLSubmittedEvent": "${CCD_DEF_COS_URL}/petition-updated",
+    "CallBackURLSubmittedEvent": "${CCD_DEF_COS_URL}/aos-submitted",
```

Comment:
Has this been agreed with the business? The 2 endpoints do different things
This change means respondent will get notifications but petitioner will stop getting any

Reply:
After agreeing with business, the new petitioner templates are added for the /aos-received callback to handle missing scenarios
https://github.com/hmcts/div-case-orchestration-service/pull/279

Reply:
PR-279 is signed off and merged. So, this can go into config now